pages/search/index.tsx

```diff
@@ -58,7 +73,9 @@ export default function Search({ results }) {
           <>
             <Heading level="h2" mb="xl" size="heading4">
               {`Displaying ${
-                totalResults > 50 ? "1-50" : totalResults.toLocaleString()
+                totalResults > RESULTS_PER_PAGE
+                  ? `${resultsStart}-${resultsEnd}`
+                  : totalResults.toLocaleString()
               } of ${totalResults.toLocaleString()} results for keyword "${
```

Comment:
potentially out of scope for this, but should "keyword" in this string be dynamic to whatever search query was made?

Reply:
Also, this string is becoming unwield. Could it be replaced with a variable and a util to generate it?